### PR TITLE
Baseline OrgLevel PCA controls refinement

### PR DESCRIPTION
**Pull request by t-rbavdekar**
…zation.json

	modified:   src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json
	modified:   src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1
	modified:   src/AzSK.ADO/Framework/Helpers/AdministratorHelper.ps1

## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 [hunk 1]
```diff
@@ -1763,106 +1763,101 @@ class Organization: ADOSVTBase
...
+                    $controlResult.AddMessage([VerificationResult]::Passed,"Number of human administrators configured meet the minimum required administrators count: $($this.ControlSettings.Organization.MinPCAMembersPermissible)");
+                }
+                [AdministratorHelper]::PopulatePCAResultsToControl($humanAccounts, $svcAccounts, $controlResult)
             }
-
-            if (($SvcAndHumanAccounts.serviceAccount | Measure-Object).Count -gt 0) {
-                $svcAccounts = $SvcAndHumanAccounts.serviceAccount | Select-Object displayName, mailAddress
-                $controlResult.AddMessage("`nService Account Administrators: $(($svcAccounts| Measure-Object).Count)", $svcAccounts)
-                $controlResult.SetStateData("List of service account Project Collection Administrators: ",$svcAccounts)
+            #ToDo Add warning that control was evaluated without graph access
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:1788]: Todo?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 [hunk 2]
```diff
@@ -1763,106 +1763,101 @@ class Organization: ADOSVTBase
...
+                    $controlResult.AddMessage([VerificationResult]::Passed,"Number of human administrators configured are within the approved limit: $($this.ControlSettings.Organization.MaxPCAMembersPermissible)");
+                }
+                [AdministratorHelper]::PopulatePCAResultsToControl($humanAccounts, $svcAccounts, $controlResult)
             }
-
-            if (($SvcAndHumanAccounts.serviceAccount | Measure-Object).Count -gt 0) {
-                $svcAccounts = $SvcAndHumanAccounts.serviceAccount | Select-Object displayName, mailAddress
-                $controlResult.AddMessage("`nService Account Administrators: $(($svcAccounts| Measure-Object).Count)", $svcAccounts)
-                $controlResult.SetStateData("List of service account Project Collection Administrators: ",$svcAccounts)
+            #ToDo Add warning that control was evaluated without graph access
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:1839]: todo?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 [hunk 3]
```diff
@@ -1763,106 +1763,101 @@ class Organization: ADOSVTBase
...
-            if($TotalPCAMembers -gt 0){
-                $controlResult.AddMessage("Verify the following Project Collection Administrators: ")
-                $controlResult.AdditionalInfo += "Total number of Project Collection Administrators: " + $TotalPCAMembers;
-            }
-
-            if (($SvcAndHumanAccounts.humanAccount | Measure-Object).Count -gt 0) {
-                $humanAccounts = $SvcAndHumanAccounts.humanAccount | Select-Object displayName, mailAddress
-                $controlResult.AddMessage("`nHuman Administrators: $(($humanAccounts| Measure-Object).Count)", $humanAccounts)
-                $controlResult.SetStateData("List of human Project Collection Administrators: ",$humanAccounts)
+        $controlResult.VerificationResult = [VerificationResult]::Failed
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:1766]: CheckMinPCACount and CheckMaxPCACount both have same implementation until the check of controlsetting is made. Can we skip the reevaluation if already done by previous control

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 (hunk 1 above)
Comment (t-rbavdekar) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1]: Manish asked to add for Sourabh's reference.  Changed it to add more details just like Himanshu did.

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 (hunk 2 above)
Comment (t-rbavdekar) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1]: Manish asked to add for Sourabh's reference. Changed it to add more details just like Himanshu did.

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 (hunk 3 above)
Comment (t-rbavdekar) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:1768]: Skipped the SvcAndHumanAccounts api call when run by second control

**Review by Garima-msft — APPROVED** [review]

**Review by SaikumarRagala — APPROVED** [review]

**Pull request merged**